### PR TITLE
Partner's modals, product catalogue download done

### PR DESCRIPTION
**Pull request by samuelkhew**
Screenshot/gif: http://recordit.co/h7DXWhG8wP

Now only left to do this contact form.

### CODE REVIEW

File: careers.html [hunk 1]
```diff
@@ -29,12 +29,17 @@
 
         <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
             <div class="navbar-nav ml-auto">
-                <a class="nav-item nav-link" href="#">ABOUT US</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link" href="#">HISTORY</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link" href="#">CATALOGUE</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link" href="#">PARTNERS</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link active" href="#">CAREERS</a>
-                <button type="button" id="contactButton">Contact</button>
+                <a class="nav-item nav-link" href="index.html#homeJumbotronDiv">ABOUT US</a><span
```
Comment (shaunwan-surialabs) [inline careers.html:34]: i think you can just omit `index.html`. just `href="#homeJumbotronDiv"` should be able to work too?

same for the rest below

File: contactForm.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+//Set email particulars
+$recipient = "samuelkhew@gmail.com";
+$subject = "Enquiry from Jetpharma.com.my";
+$headers = 'From: samuelkhew@gmail.com' . "\r\n" .'Reply-To: samuelkhew24@gmail.com';
+
+//Send email and redirect to the homepage
+mail($recipient, $subject, $emailContent, $headers);
+
+?>
```
Comment (shaunwan-surialabs) [inline contactForm.php]: missing new line

File: myScripts.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                if (form.checkValidity() === false) {
+                    event.preventDefault();
+                    event.stopPropagation();
+                }
+                form.classList.add('was-validated');
+
+            }, false);
+        });
+    }, false);
+})();
```
Comment (shaunwan-surialabs) [inline myScripts.js]: missing new line

File: careers.html (hunk 1 above)
Comment (samuelkhew) [reply careers.html:34]: @shaunwan-surialabs just tried it... doesnt work... index.html needs to be there..

Comment (shaunwan-surialabs) [reply careers.html:34]: ok sorry my bad..i thought this was `index.html` but it's actually `career.html`

File: myScripts.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
+//Download Form: Setting the Place of Practice Variable to "N/A"
+function setPlaceVariable() {
+    if (document.getElementById('downloadYes').checked) {
```
Comment (shaunwan-surialabs) [inline myScripts.js]: you can just do a if statement to check if `downloadYes` is not checked by adding a `!` infront of a boolean (so `!true` will be `false`)

```
if (!document.getElementById('downloadYes').checked) {
   document.getElementById("downloadPlaceInput").value = "N/A";
}
```

**Pull request merged**